api-docs/graphql-api-rate-limits

```diff
@@ -0,0 +1,26 @@
+---
+title: GraphQL Rate Limits
+category: 652e3a8e279c3f001f9acdc3
+---
+
+## Query Complexity & Rate Limiting in our GraphQL API
+
+Query complexity in GraphQL refers to the measure of computational resources required to execute a specific GraphQL query. 
+It takes into account factors such as the depth of the query, the number of requested fields, and any potential nested or recursive structures. 
+Query complexity is a crucial aspect for optimizing GraphQL performance and preventing resource-intensive or potentially malicious queries from overloading a server. 
+
+By setting query complexity thresholds, Trackunit aims to establish limits on the computational load a single query can impose, helping to maintain the stability and efficiency of our GraphQL API. 
+This concept enables us to strike a balance between offering powerful and flexible queries to clients while ensuring the server's responsiveness and resource consumption remain manageable.
+
+The rate limiting rules of our GraphQL API are connected to the calculated complexity scores of your queries. 
+A GraphQL query can be arbitrarily complex, therefore we parse all incoming queries, calculate their complexity scores in points and subtract those from the rate limit for your API user.
+
+**Currently the Rate Limits for our GraphQL API are:**
+
+- A single query is limited to 50000 complexity points.)
+- All queries within a 10 minute period are limited to 500000 complexity points per API user.
+
+
+> TODO
```

Comment:
@mmt-trackunit Can you provide a short description of how the gql api responds if a rate limit is hit?